### PR TITLE
Bot can run multiple instances at the same time

### PR DESCRIPTION
**Pull request by StopmotionSimonYT**
# Description
With this pull request you can use the program multiple times at the same time. This is because the temp folder is determined per folder. For example: if the video id is 12356, then the temp folder is assets/temp/123456/png . If you run another instance with the id 67890 then the temp folder is assets/temp/67890/png .

# Issue Fixes
None

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [x] I am using the recommended development environment
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have formatted and linted my code using python-black and pylint
- [x] I have cleaned up unnecessary files
- [x] My changes generate no new warnings
- [x] My changes follow the existing code-style
- [x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

None

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -61,8 +69,6 @@ def run_many(times):
 
 
 def shutdown():
-    print_markdown("## Clearing temp files")
-    cleanup()
```
Comment (CallumIO) [inline main.py:65]: Why was this removed?

File: main.py [hunk 2]
```diff
@@ -36,20 +37,27 @@
...
     length, number_of_comments = save_text_to_mp3(reddit_object)
     length = math.ceil(length)
     download_screenshots_of_reddit_posts(reddit_object, number_of_comments)
     bg_config = get_background_config()
     download_background(bg_config)
-    chop_background_video(bg_config, length)
+    chop_background_video(bg_config, length, reddit_object)
     make_final_video(number_of_comments, length, reddit_object, bg_config)
 
+def id(reddit_obj: dict):
```
Comment (CallumIO) [inline main.py:54]: This function should probably placed in the utils folder, and omitting the print statement

All future needs for the id can then use this function

**Review by CallumIO — CHANGES_REQUESTED** [review]
Looks good, just those few things that could be improved

File: main.py (hunk 1 above)
Comment (StopmotionSimonYT) [reply main.py:65]: Because if the shutdown action was called before the reddit id was determined, it deletes the whole temp folder.

File: main.py (hunk 2 above)
Comment (StopmotionSimonYT) [reply main.py]: I have put it in the utils folder

File: main.py (hunk 1 above)
Comment (StopmotionSimonYT) [reply main.py:65]: I could make it that it would remove the whole temp folder with all the sessions but that could affect current other sessions.

Comment (CallumIO) [reply main.py:65]: > Because if the shutdown action was called before the reddit id was determined, it deletes the whole temp folder.

Could add a check to see if id is null, if it is, don't remove anything

Comment (StopmotionSimonYT) [reply main.py:65]: Good idea! Haven't thought about that.I'm working on something else now, in a few hours I'll update it.

File: video_creation/final_video.py [hunk 3]
```diff
@@ -139,26 +140,26 @@ def make_final_video(
     #    # lowered_audio = audio_background.multiply_volume( # todo get this to work
     #    #    VOLUME_MULTIPLIER)  # lower volume by background_audio_volume, use with fx
     #    final.set_audio(final_audio)
-    final = Video(final).add_watermark(
+    '''final = Video(final).add_watermark(
         text=f"Background credit: {background_config[2]}", opacity=0.4
-    )
+    )''' #  todo add watermark
```
Comment (JasonLovesDoggo) [inline video_creation/final_video.py:145]: You got to finish those todo's...

**Review by JasonLovesDoggo — CHANGES_REQUESTED** [review]

File: video_creation/final_video.py (hunk 3 above)
Comment (StopmotionSimonYT) [reply video_creation/final_video.py:145]: You're right, I forgot it because it was difficult to do. With the commit I just did it is added.

**Comment by JasonLovesDoggo** [comment]
> Any reason this isn't merged yet?

Waiting for @callumio s review

**Review by callumio — APPROVED** [review]

**Comment by MarkelAdanez** [comment]
It keeps makes the same video in the different instace, is this only me?
